timesketch/api/v1/resources/sigma.py

```diff
@@ -87,15 +125,274 @@ def get(self, rule_uuid):
                     return_rule = rule
 
         if return_rule is None:
-            abort(HTTP_STATUS_CODE_NOT_FOUND, "No sigma rule found with this ID.")
+            abort(
+                HTTP_STATUS_CODE_NOT_FOUND, "No sigma rule found with this ID."
+            )
 
-        meta = {"current_user": current_user.username, "rules_count": len(sigma_rules)}
+        meta = {
+            "current_user": current_user.username,
+            "rules_count": len(sigma_rules),
+        }
         return jsonify({"objects": [return_rule], "meta": meta})
 
 
+# TODO(jaegeral): deprecate this class
 class SigmaByTextResource(resources.ResourceMixin, Resource):
     """Resource to get a Sigma rule by text."""
 
+    @login_required
+    def post(self):
+        """Handles POST request to the resource.
+        Returns:
+            JSON sigma rule
```

Comment:
I see the docstrings for these endpoints are still a bit vague. Can we make them more explicit?